generics/helpers/utils.js

```diff
@@ -3,363 +3,390 @@
+ * @function
+ * @name lowerCase
+ * @param {String} str
+ * @returns {String} returns a lowercase string. ex: HELLO, o/p: hello
+ */
 
 function lowerCase(str) {
-  return str.toLowerCase()
+	return str.toLowerCase()
 }
 
- /**
-  * check whether the given string is url.
-  * @function
-  * @name checkIfStringIsUrl - check whether string is url or not.
-  * @param {String} str 
-  * @returns {Boolean} returns a Boolean value. ex:"http://example.com:3000/pathname/?search=test" , o/p:true
-*/
+/**
+ * check whether the given string is url.
+ * @function
+ * @name checkIfStringIsUrl - check whether string is url or not.
+ * @param {String} str
+ * @returns {Boolean} returns a Boolean value. ex:"http://example.com:3000/pathname/?search=test" , o/p:true
+ */
 
 function checkIfStringIsUrl(str) {
-  var pattern = new RegExp('^(https?:\\/\\/)?' + // protocol
-    '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.?)+[a-z]{2,}|' + // domain name
-    '((\\d{1,3}\\.){3}\\d{1,3}))' + // OR ip (v4) address
-    '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' + // port and path
-    '(\\?[;&a-z\\d%_.~+=-]*)?' + // query string
-    '(\\#[-a-z\\d_]*)?$', 'i'); // fragment locator
-  return pattern.test(str);
+	var pattern = new RegExp(
+		'^(https?:\\/\\/)?' + // protocol
+			'((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.?)+[a-z]{2,}|' + // domain name
+			'((\\d{1,3}\\.){3}\\d{1,3}))' + // OR ip (v4) address
+			'(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' + // port and path
+			'(\\?[;&a-z\\d%_.~+=-]*)?' + // query string
+			'(\\#[-a-z\\d_]*)?$',
+		'i'
+	) // fragment locator
+	return pattern.test(str)
 }
 
- /**
-  * Parse a single column.
-  * @function
-  * @name valueParser - Parse value
-  * @param {String} dataToBeParsed - data to be parsed. 
-  * @returns {Object} returns parsed data
-*/
+/**
+ * Parse a single column.
+ * @function
+ * @name valueParser - Parse value
+ * @param {String} dataToBeParsed - data to be parsed.
+ * @returns {Object} returns parsed data
+ */
 
 function valueParser(dataToBeParsed) {
+	let parsedData = {}
 
-  let parsedData = {}
+	Object.keys(dataToBeParsed).forEach((eachDataToBeParsed) => {
+		parsedData[eachDataToBeParsed] = dataToBeParsed[eachDataToBeParsed].trim()
+	})
 
-  Object.keys(dataToBeParsed).forEach(eachDataToBeParsed => {
-    parsedData[eachDataToBeParsed] = dataToBeParsed[eachDataToBeParsed].trim()
-  })
+	if (parsedData._arrayFields && parsedData._arrayFields.split(',').length > 0) {
+		parsedData._arrayFields.split(',').forEach((arrayTypeField) => {
+			if (parsedData[arrayTypeField]) {
+				parsedData[arrayTypeField] = parsedData[arrayTypeField].split(',')
+			}
+		})
+	}
 
-  if(parsedData._arrayFields && parsedData._arrayFields.split(",").length > 0) {
-    parsedData._arrayFields.split(",").forEach(arrayTypeField => {
-      if (parsedData[arrayTypeField]) {
-        parsedData[arrayTypeField] = parsedData[arrayTypeField].split(",")
-      }
-    })
-  }
+	return parsedData
+}
+
+function getGotenbergConnection() {
```

Comment:
@MallanagoudaB Why this is added in utils ? create a new file in generics/services/gotenberg.js and add this function there

Reply:
it was there before and have the same formate in ML-REPORT